tests/ecosystem/upgrade/test_upgrade.py

```diff
@@ -81,10 +82,15 @@ def test_check_mon_pdb_post_upgrade():
     mons_after_upgrade = ceph_obj.get_mons_from_cluster()
     log.info(f"Mons after upgrade {mons_after_upgrade}")
 
-    disruptions_allowed, min_available_mon = get_mon_pdb()
+    disruptions_allowed, min_available_mon, max_unavailable_mon = get_mon_pdb()
     log.info(f"Number of Mons Disruptions_allowed {disruptions_allowed}")
     log.info(f"Minimum_available mon count {min_available_mon}")
+    log.info(f"Minimum_available mon count {max_unavailable_mon}")
 
     # The PDB values are considered from OCS 4.5 onwards.
     assert disruptions_allowed == 1, "Mon Disruptions_allowed count not matching"
-    assert min_available_mon == 2, "Minimum available mon count is not matching"
+    if get_ocs_parsed_version() <= 4.6:
```

Comment:
This is not good solution. See the issue here:
https://github.com/red-hat-storage/ocs-ci/issues/4066

I didn't have time to start working on the issue in general. Once I do I will probably cover the function get_ocs_parsed_version to return semantic version. But for now please handle as suggested in the issue.

Reply:
using semantic version now